setup.py

```diff
@@ -2,37 +2,37 @@
 
 dependencies = [
     "multidict==5.1.0",  # Avoid 5.2.0 due to Avast
-    "aiofiles==0.7.0",  # Async IO for files
-    "blspy==1.0.8",  # Signature library
-    "chiavdf==1.0.3",  # timelord and vdf verification
-    "chiabip158==1.0",  # bip158-style wallet filters
-    "chiapos==1.0.7",  # proof of space
+    "aiofiles==0.8.0",  # Async IO for files
+    "blspy==1.0.9",  # Signature library
+    "chiavdf==1.0.4",  # timelord and vdf verification
+    "chiabip158==1.1",  # bip158-style wallet filters
+    "chiapos==1.0.8",  # proof of space
     "clvm==0.9.7",
     "clvm_rs==0.1.17",
     "clvm_tools==0.4.3",
-    "aiohttp==3.7.4",  # HTTP server for full node rpc
+    "aiohttp==3.8.1",  # HTTP server for full node rpc
     "aiosqlite==0.17.0",  # asyncio wrapper for sqlite, to store blocks
     "bitstring==3.1.9",  # Binary data management library
     "colorama==0.4.4",  # Colorizes terminal output
-    "colorlog==5.0.1",  # Adds color to logs
+    "colorlog==6.6.0",  # Adds color to logs
     "concurrent-log-handler==0.9.19",  # Concurrently log and rotate logs
     "cryptography==3.4.7",  # Python cryptography library for TLS - keyring conflict
     "fasteners==0.16.3",  # For interprocess file locking
     "keyring==23.0.1",  # Store keys in MacOS Keychain, Windows Credential Locker
     "keyrings.cryptfile==1.3.4",  # Secure storage for keys on Linux (Will be replaced)
     #  "keyrings.cryptfile==1.3.8",  # Secure storage for keys on Linux (Will be replaced)
     #  See https://github.com/frispete/keyrings.cryptfile/issues/15
-    "PyYAML==5.4.1",  # Used for config file format
+    "PyYAML==6.0",  # Used for config file format
     "setproctitle==1.2.2",  # Gives the chia processes readable names
     "sortedcontainers==2.4.0",  # For maintaining sorted mempools
     "websockets==8.1.0",  # For use in wallet RPC and electron UI
-    "click==7.1.2",  # For the CLI
+    "click==8.0.3",  # For the CLI
```

Comment:
I have run into issue with Click 8.  They may not be so applicable here but I would want to have thought through them first.  The one particular thing I ran into was that Windows expects programs to handle command line parsing including shell expansion where as in Linux (and macOS I believe) the shell is expected to do expansion and the process receives a 'list' of the arguments.  If I remember correctly, Click started respecting this difference which means it expands wildcards and processes quoting that it did not before.

```suggestion
    "click==7.1.2",  # For the CLI
```